workflows/fastquorum.nf

```diff
@@ -41,19 +41,41 @@ workflow FASTQUORUM {
     ch_versions = Channel.empty()
     ch_multiqc_files = Channel.empty()
 
+    // TODO: This won't work for single-ends!
```

Comment:
we have to be careful as we may have anywhere from one (1) to four (4) FASTQs per sample: any of index1 (i7), index2 (i5), read1 (R1), and read2 (R2), so if we want to support concatenating FASTQs, we'll need to handle all those cases.